tests/acceptance/test_dbus.py

```diff
@@ -29,7 +29,7 @@
     },
     "InventoryPollIntervalSeconds": 5,
     "RetryPollIntervalSeconds": 5,
-    "ServerCertificate": "/usr/share/doc/mender-client/examples/demo.crt",
+    "ServerCertificate": "/usr/local/share/ca-certificates/mender/server.crt",
```

Comment:
This should be removed instead. It should now work without a `ServerCertificate` entry.

Reply:
Right!